ngonoessono_a05a.js

```diff
@@ -0,0 +1,44 @@
+/*Gertrude Vanessa Ngono Essono
+ngonoessono_a05a.js
+INFO2124
+Thoendel
+01/172020*/
+
+const todo = [];
+todo[0]= 'cleaning';
+todo[1]= 'laundry';
+todo[2]= 'shopping';
+todo[3]= 'volunteering';
+console.log(todo);
+todo.unshift('painting');
+console.log(todo);
+
+todo.reverse();
+console.log(todo);
+
+todo.sort();
+console.log(todo);
+
+todo.includes('x');
```

Comment:
Good job! You've populated the array, added to the beginning, reversed the order and found if x is in the array.
👍